testdata/b1/README.md

```diff
@@ -14,6 +14,28 @@ $ openssl x509 -req -in server.csr -CA ca.cert -CAkey ca.privkey -CAcreateserial
 $ cat server.cert ca.cert >fullchain.cert
 ```
 
+OCSP:
+
+Make sure your openssl.cfg has an ocsp extension section, like:
+
+[ ocsp ]
```

Comment:
Put this and other code snippets in ``` quotes ``` like above. (Click View button to preview the markdown rendering.)

Reply:
Done